src/PrestaShopBundle/Form/Admin/Type/DeltaQuantityType.php

```diff
@@ -42,6 +43,11 @@
  */
 class DeltaQuantityType extends TranslatorAwareType
 {
+    /**
+     * this is the biggest int number that can be saved in database, bigger than this will throw error
+     */
+    public const INT_32_MAX = 2147483647;
```

Comment:
It kind of bothers me that this value is duplicated in two places, it would be nice to find a common place where this should be placed. Usually, we add this kind of const on VO objects related to the feature. But we don't have a VO that holds the quantity, so I'm hesitating between `PrestaShop\PrestaShop\Core\Domain\Product\Stock\ValueObject|StockId` or `PrestaShop\PrestaShop\Core\Domain\Product\Stock\ValueObject\StockModification`. Or maybe directly in `StockAvailable` but I'm not a fan of relying on a const placed in a legacy object...